generators/directive/index.js

```diff
@@ -1,14 +1,19 @@
 'use strict';
 
-var Generator = require('yeoman-generator'),
-    chalk     = require('chalk'),
-    yosay     = require('yosay'),
-    fs        = require('fs');
+/**
+ * Generates a directive.
+ */
+
+var Generator  = require('yeoman-generator'),
+    chalk      = require('chalk'),
+    yosay      = require('yosay'),
+    caseIt     = require('change-case'),
```

Comment:
may as well utilize a package for casing strings. it's got pretty much every function we might want.